source/DmapSupport.cpp

```diff
@@ -184,37 +184,51 @@ HRESULT GetControllerBaseAddress(PWCHAR deviceName, HANDLE & handle, PVOID & bas
                         g_openDeviceMask |= 1 << i;
 
                         IOControlCode^ IOCTL = ref new IOControlCode(0x423, 0x100, IOControlAccessMode::Any, IOControlBufferingMethod::Buffered);
```

Comment:
If this is done, make sure it is done in a way that is compatible with the driver as well as the user-mode code.